docs/source/contributing/typescript.md

```diff
@@ -9,14 +9,126 @@ myst:
 
 # TypeScript
 
+TypeScript support (5.2.2) has been added to Volto core.
```

Comment:
I don't think we want to specify a version of third party software that will need to be updated with each release. I am lazy. 😉 

````suggestion
```{versionadded} 17.0.0-alpha.22
```
TypeScript support has been added to Volto core.
````